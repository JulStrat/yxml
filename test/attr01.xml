<a AttributeName='val' Ot-he_r:4tt.r = " val	 &#x009;&#9;&#009;" />
