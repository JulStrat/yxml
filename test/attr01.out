
elemstart a
attrstart AttributeName
data val
attrend
attrstart Ot-he_r:4tt.r
data  val  \x09\x09\x09
attrend
content
elemend
ok
